### DISCUSSION TITLE
Hex color from makeStyles

### DISCUSSION THREAD

**Discussion by rexkenley** [microsoft/fluentui#31150] [Q&A]
How do I get the hex color code from the makeStyles hook object?
https://react.fluentui.dev/?path=/docs/concepts-developer-styling-components--page#applying-theme-to-styles

const useStyles = makeStyles({
  primary: { color: tokens.colorBrandForeground1 }
});
 
themeStyle = useStyles();
  console.log(`Primary : ${themeStyle.primary}`);

I am getting this result
`Primary : ___1vvfegh_0000000 f16muhyy`

**Comment by pradeept95** [comment]
Hi @rexkenley , You can't because it uses a scoped CSS variable pattern to support build time CSS calculation, but this might be helpful, please check following discussion

https://github.com/microsoft/fluentui/discussions/30260

**Comment by rexkenley** [comment]
https://github.com/microsoft/fluentui/discussions/31171
